packages/js/email-editor/src/components/personalization-tags/rich-text-utils.tsx

```diff
@@ -0,0 +1,191 @@
+import * as React from '@wordpress/element';
```

Comment:
It is just a detail. I think this file could be `rich-text-utils.ts` because there is no jsx syntax.